internal/api/client.go

```diff
@@ -17,25 +20,50 @@ type ClientConfig struct {
 	AccessToken      string
 	OrganizationSlug string
 	ServerBaseUrl    string
+	DebugEnabled     bool
 }
 
 // authTransport is a middleware for the HTTP client.
 type authTransport struct {
-	accessToken string
+	accessToken  string
+	debugEnabled bool
 }
 
 // RoundTrip adds the Authorization header to all requests made by the HTTP client.
 func (t *authTransport) RoundTrip(req *http.Request) (*http.Response, error) {
+	if t.debugEnabled {
+		fmt.Println("--- 🕵️ DEBUG")
```

Comment:
One other thing: consider using `log.Print{,f,ln}` or `fmt.Fprint{,f,ln}(os.Stderr, ...)` to send debugging logs to a separate stream than the normal output.